### ISSUE TITLE
No longer maintained?

### ISSUE THREAD

**Issue by TylerRick** [nikhaldi/exception_notification-rake#23]
Hi! Is this project still maintained? I see there are PRs from 2016 that are still unresolved.

![image](https://user-images.githubusercontent.com/14151/58043567-5ea42200-7af2-11e9-90d9-6f7250b46e2b.png)

**Comment by lostapathy** [comment]
@nikhaldi I would be more than happy to help maintain this gem if you would like to offload/share responsibility for it.

**Comment by nikhaldi** [comment]
Hey! I have not actively maintained this in the last few years because I sadly haven't been using Ruby. So I'm super rusty and out of touch with the ecosystem and don't necessarily trust myself to review PRs.

@lostapathy thanks for stepping up, help would be very appreciated. For the moment, maybe have a quick look at the PRs that @TylerRick submitted and add comments if you have any? I'm happy to merge these tomorrow and put out a release. We can talk about making you an official maintainer as well somehow.

**Comment by lostapathy** [comment]
#22 is mine and just a travis change, it's good to go.  I will comment on the other PRs directly.

**Comment by nikhaldi** [comment]
Thanks a lot @lostapathy! Recent PRs are now merged except for https://github.com/nikhaldi/exception_notification-rake/pull/27 - tbh I have to spend some time on that one to determine if it makes sense. If you have opinions, let us know.

Even without that one I think we are good for a release (especially to get the 4.3 support). Any additional changes that would be good? Otherwise I'll put out a release candidate.

**Comment by lostapathy** [comment]
I agree with your comment on #27 - rebasing that to master will make it easier to follow.

FWIW I pushed this to production at work based on #25's branch and 4.3 enabled about 10 hours ago.  I will update it to point at master on this tree tomorrow morning.

**Comment by TylerRick** [comment]
#27 is still ready to merge
